packages/init/src/index.ts

```diff
@@ -15,11 +16,17 @@ const CONFIG_PREFIX = '--force';
  */
 
 export default function initializeInquirer(...args: string[]): Function | void {
-    const packages = args;
-    const includesDefaultPrefix = packages.includes(AUTO_PREFIX);
-    const generateConfig = packages.includes(CONFIG_PREFIX);
-    if (packages.length === 0 || includesDefaultPrefix || generateConfig) {
+    const cli = new WebpackCLI();
+    const { opts, unknownArgs } = cli.argParser(options, args, true);
```

Comment:
Why do not throw error inside `cli.argParser` on unknown flags?

Reply:
`serve` supports the use of standalone flags, `webpack-cli serve --port <port> --hot`. If we were to handle `unknownArgs` within `arg-parser` standalone flags (when used with `serve`) won't be recognized by default.
https://github.com/webpack/webpack-cli/blob/f1876c125be90e0b0ae96e6e9dbaf91ed447c779/packages/serve/src/index.ts#L17-L20

When it comes to `init`, we've to filter out the `cli-flags` from `unknownArgs`, or else `webpack-scaffold-<name>` would be treated as an unknown argument.

Also, handling unknown args within `arg-parser` seems to interfere with the standalone flags.